whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -156,6 +161,75 @@ public Object mapSearch(final String requestUrl, final List<RpslObject> objects,
         return mapCommons(searchResult, requestUrl);
     }
 
+    public Object mapOrganisationEntity(final String requestUrl, Iterator<RpslObject> objects,
+                                        final RpslObjectDao objectDao, final int maxResultSize){
+
+        final List<Autnum> autnums = Lists.newArrayList();
+        final List<Ip> networks = Lists.newArrayList();
+        RdapObject organisation = new RdapObject();
+        for (Iterator<RpslObject> it = getTopLevelObjects(objects); it.hasNext(); ) {
+            final RpslObject object = it.next();
```

Comment:
why we are passing org objects as well to toplevelObjects? It should just filter the top level resources . Can we just pass on the inetnum and inet6num to this function and later on add it in rdap networks setter?